### PR TITLE
Save and load JSON locally using NSUserDefaults.

### PR DESCRIPTION
**Pull request by JeffGuKang**
Save and Load locally are useful functions to remain just before status for iOS applications. When the application is terminated.  For that reason,  I added the functions for SwiftyJSON.

### CODE REVIEW

File: Source/SwiftyJSON.swift [hunk 1]
```diff
@@ -1376,3 +1376,30 @@ func >=(lhs: NSNumber, rhs: NSNumber) -> Bool {
         return lhs.compare(rhs) != NSComparisonResult.OrderedAscending
     }
 }
+
+
+//MARK: - Save and load JSON locally using NSUserDefaults.
+extension JSON {
+    public func saveToLocal(forKey: String) -> Bool {
```
Comment (kandelvijaya) [inline Source/SwiftyJSON.swift]: This is a good use case. Can we make the function name more readable by doing so ```saveToLocal``` -> ```saveLocally``` and the same for loadLocally

Comment (JeffGuKang) [inline Source/SwiftyJSON.swift]: I think that is better name. Updating.

**Comment by zhigang1992** [comment]
IMHO, this is more like a plugin than a part SwiftyJSON.

Might be we should create a new project for this. 😄

**Comment by zhigang1992** [comment]
Cause user might want to save to somewhere else other than NSUserDefaults, even if it's NSUserDefaults, they might not want to put it in the standardDefaults.

But as a more concrete plugin, this will make sense.

I'll close this PR for now, cause I kinda had to clean this repo up a bit.

But feel free to reopen it if you'd like to discuss more.

Cheers

**Pull request closed without merging**